src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -29,6 +29,7 @@
 import us.kbase.mobu.util.ProcessHelper;
```

Comment:
Have you tested with mac & linux, and do you see any of the issues described in the `--user` ticket?

Reply:
Running as root causes any files you create in the container to be owned by "root" in the host. To me this is not a problem and is something I would expect with docker. But Shane said it was a usability issue for people. I'll to talk to Shane about it soon in person.

Reply:
What happens if you run the container as root and produce files owned by root on a system where you don't have root access? Seems like you wind up with a bunch of files in your home directory you can't touch.